README.md

```diff
@@ -15,84 +15,107 @@ After you have all the prerequisites installed and have configured your
 [`testnet.toml`](./testnet.toml) file appropriately:
 
 1. Set up your [personal access token for DO](https://docs.digitalocean.com/reference/api/create-personal-access-token/)
-  ```bash
-  doctl auth init
-  ```
+
+    ```bash
+    doctl auth init
+    ```
+
+    If you have previously used another token, you may need to force the use of the one you just generated here.
```

Comment:
Could you add something like "In this case, you might be able to skip directly to step 5."